src/functions-and-arrays.js

```diff
@@ -1,20 +1,74 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers (a, b) {
+  if (a > b) {
+      return a;
+  }
+  else if (a < b) {
+      return b;
+  }
+  else {
+    return a;
+  }
+}
 
 // Iteration #2: Find longest word
+function findLongestWord (array) {
+  let length = 0;
+  let longestWord;
+  for (let i = 0; i < array.length; i++) {
+      if (array[i].length > length) {
+          length = array[i].length;
+          longestWord = array[i]; 
+      }
+      return longestWord;
```

Comment:
here you should put the `return longestWord;` out of the "for loop", so it should be between the line 24 and 25.